File: Factorial.py
f = 1
n = int(input("Enter a number: "))
if n < 0:
    print("Factorial of negetive numbers do not exist")
elif n == 0 or n == 1:
    print("Factorial of", n, "= 1")
else:
    for i in range(1, n + 1):
        f *= i
    print("Factorial of", n, "=", f)

File: Factorial.cpp
#include<stdio.h>
int main()
{
	int n;
	int f=1;
	printf("Enter a number:\n");
	scanf("%d",&n);
	if(n>1)
	{
		for(int i=1;i<=n;i++)
		f=f*i;
		printf("Factorial of %d is: %d\n",n,f);
	}
	else if(n==0||n==1)
	printf("The Factorial of %d is: %d\n",n,f);
	return 0;
}